packages/lambda-tiler/src/routes/__tests__/wmts.test.ts

```diff
@@ -107,5 +107,11 @@ o.spec('WMTSRouting', () => {
       '<ows:Title>Google Maps Compatible for the World</ows:Title>',
       '<ows:Title>LINZ NZTM2000 Map Tile Grid V2</ows:Title>',
     ]);
+
+    const resourceURLs = lines.filter((f) => f.includes('<ResourceURL')).map((f) => f.trim());
+    o(resourceURLs).deepEquals([
+      '<ResourceURL format="image/png" resourceType="tile" template="https://tiles.test/v1/tiles/aerial/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png?api=d01f7w7rnhdzg0p7fyrc9v9ard1&amp;config=Q5pC4UjWdtFLU1CYtLcRSmB49RekgDgMa5EGJnB2M&amp;date%5Bbefore%5D=2021" />',
+      '<ResourceURL format="image/png" resourceType="tile" template="https://tiles.test/v1/tiles/ōtorohanga-urban-2021-0.1m/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png?api=d01f7w7rnhdzg0p7fyrc9v9ard1&amp;config=Q5pC4UjWdtFLU1CYtLcRSmB49RekgDgMa5EGJnB2M&amp;date%5Bbefore%5D=2021" />',
```

Comment:
We shouldn't use a date range on a individual imagery set.

Reply:
I see, I just think this ResourceURL is missing the daterange parameter so I made this change. I think WMTS already contain the `filteredLayers`, then we don't need this changes. The PR could be closed.